cernopendata_client/validator.py

```diff
@@ -9,12 +9,13 @@
 """cernopendata-client input validation methods."""
 
 import click
+import sys
 
 from .printer import display_message
 
 
 def validate_recid(recid=None):
-    """Return True if OK, raises click.BadParameter exception otherwise.
+    """Return True if record id is valid.
```

Comment:
You can say *"Return True if record ID is valid, exit otherwise."* so that users won't be surprised by the exit behaviour instead of returning False.

Reply:
Done 👍